tests/docs/synced-array-callback-reference.test.sh

```
@@ -70,7 +76,11 @@ require_text "$SYNC_EXAMPLES" 'same length, reassigning the array, or changing i
 require_text "$TROUBLESHOOTING" '### Synced Array Callback Silent Failure'
 require_text "$TROUBLESHOOTING" 'same-length element changes, array reassignments, and array length changes'
 require_text "$TROUBLESHOOTING" 'ApplyValues()'
+require_text "$TROUBLESHOOTING" 'late joiner'
+require_text "$TROUBLESHOOTING" 'baseline'
 require_text "$CHEATSHEET" 'Array contents changed: use `OnDeserialization()`'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)